src/dmd/aggregate.d

```diff
@@ -193,7 +193,7 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
             if (v.aliassym)
                 return 0;   // If this variable was really a tuple, skip it.
 
-            if (v.storage_class & (STC.static_ | STC.extern_ | STC.tls | STC.gshared | STC.manifest | STC.ctfe | STC.templateparameter))
+            with (STC) if (v.storage_class & (static_ | extern_ | tls | gshared | manifest | ctfe | templateparameter))
```

Comment:
I have a suspicion Walter won't like this.

Reply:
Okay, cancelled this change.